.gitignore

```diff
@@ -82,6 +82,7 @@ expected/*
 !/sound/**/*custom*/**/*.aiff
 !/assets/**/*custom*.bin
 !/assets/**/*custom*/**/*.bin
-
+tools/*.md
 # Doxygen
-docs/html/*
+tools/doxygen_syms.md
+tools/doxygen_syms.md
```

Comment:
Please revert this change. We need doxygen_syms generated in the PRs for the github pages API site.

Reply:
Yeah VSCode autmatically did that. Reverting now

Reply:
@MegaMech can you confirm that this reverted? It's not showing right for me